obs-studio-server/source/nodeobs_api.cpp

```diff
@@ -1453,14 +1464,20 @@ double OBS_API::getDroppedFramesPercentage(void)
 	return percent;
 }
 
-double OBS_API::getCurrentBandwidth(void)
+void OBS_API::OutputStats::getCurrentStats(bool recording)
```

Comment:
In my opinion it would be easier to return `kbitsPerSec` and `dataOutput` here rather than using global variables.

Reply:
I would like to keep it this way because the storing and calculations of `kbitsPerSec` and `dataOuput` are all handled by the structure `OutputStats`. It is better organized this way to have just two objects: one for the streaming output and another for the recording output. I agree that there is no need to have `streamingOutputStats` and `recordingOutputStats` be global variables so I moved them to be static member variables of the `OBS_API` class

Reply:
Ok I see, I would do 2 things then:
 1. Remove `getCurrentStats` from being a function inside the `OutputStats`, would be clearer to put it as a static function.
 2. Remove `bool recording` is the first argument and put `obs_output_t *output` instead. Then inside the function, make sure the output is not null.

Reply:
Sure thing, will get to make those changes

Reply:
Changes done. I also renamed `getCurrentStats` to `getCurrentOutputStats`